VCFv4.3.tex

```diff
@@ -536,29 +565,36 @@ \section{INFO keys used for structural variants}
 ##INFO=<ID=END,Number=1,Type=Integer,Description="End position of the variant described in this record">
 \end{verbatim}
 \normalsize
-For precise variants, END is POS + length of REF allele - 1, and the for imprecise variants the corresponding best estimate.
+For precise variants, END is POS + length of REF allele - 1, and the for imprecise variants, END is the corresponding best estimate.
```

Comment:
I'm not sure if this PR is still active, but it might be good to further clarify the use of the `END` key. I've noticed a few cases where, for interchromosomal BNDs, tools are storing the second chromosome in the `CHR2` key and the position on the second chromosome in the `END` key. I think that Delly might do this, or did it in a previous version, as do VCFs produced by the current version of the Talkowski Lab's SV pipeline. This breaks other tools which want to produce the interval for the variant by looking at `END`. In particular htsjdk fails on these files. If the variant is an imprecise BND, I can see how this interpretation of the spec arose, but I think it's worth a clarification on whether this should be allowed or not. Personally I think it should not be, and the spec should clarify that the value in `END` should always be on the same contig as `CHR` and come after `POS`. 

If it's a topic worthy of its own issue, I could create a separate ticket/PR to try to address it.

Also, there's a typo in this sentence both the current version and the proposed change (`and the for`).

Reply:
I support this. Ideally there should be only one meaning of the END tag and it makes sense to restrict it to the same chromosome. That was also the original intention.

Reply:
The spec already has `CIPOS` for `BND` events. The `END` clarification was intended to indicate what happens to the RHS breakend for symbolic non-bnd events. Technically speaking, even with this change you still can't encode an imprecise `<DEL>` with different bounds intervals for the LHS and RHS breakends and have to use `BND`.

We should indeed be explicit in disallowing `END` for `BND` events.